OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.Validations.Remove.cs

```diff
@@ -19,7 +19,7 @@ public async Task ShouldThrowValidatonExceptionOnRemoveWhenAssignmentIdIsInvalid
         {
             // given
             Guid randomAttachmentId = Guid.NewGuid();
-            Guid randomAssignmentId = default;
+            Guid randomAssignmentId = Guid.Empty;
```

Comment:
the variable name to be 'invalidAssignmentId'

Reply:
Replaced "random" with "invalid" and pushed changes.